README.md

```diff
@@ -36,9 +36,21 @@ This tap:
 - Outputs the schema for each resource
 - Incrementally pulls data based on the input state
 
-## Limitations
+## Running the tap
 
- - Only designed to work with Xero [Partner Applications](https://developer.xero.com/documentation/auth-and-limits/partner-applications), not Private Applications.
+- Create config file with the following variables:
+    - consumer_key
+    - client_secret
+    - start_date
+- Validate that the private key is located at the following path: `~/.ssh/privatekey.pem`
```

Comment:
What private key? If I were running this would I need to generate one of these?

Reply:
this is the application's key generated from Xero. I will add a comment to clarify this